user-group/survey01-email.html

```diff
@@ -0,0 +1,114 @@
+											<table style="border-spacing:0;" width="100%">
+												<tbody>
+													<tr>
+														<td style="text-align:center;padding-top:0;padding-bottom:0;padding-right:60px;padding-left:60px;vertical-align:top">
+															<h3 style="font-family:sans-serif;font-weight:lighter;text-align:center;font-size:32px;line-height:28px;Margin-top:0;Margin-bottom:10px;color:#00c0cb;"><span style="color:#00C0CB">Take the User Group Survey</span></h3>
+															<h4 style="font-family:sans-serif;font-weight:lighter;text-align:center;font-size:22px;line-height:28px;Margin-top:0;Margin-bottom:30px;color:#ffffff;"><span style="color:white">Help us understand how your institution supports accessibility with 12 questions.</span></h4>
+														</td>
+													</tr>
+													<tr>
+														<td style="text-align:center;padding-top:0;padding-bottom:0;padding-right:60px;padding-left:60px;vertical-align:top;height:80px">
+															<a href="[SurveyLink]" target="_blank" style="padding-top:10px;padding-bottom:10px;padding-right:20px;padding-left:20px;background-color:#007900;font-family:sans-serif;font-weight:lighter;font-size:22px;line-height:22px;text-align:left;text-decoration: none;color:#ffffff;">Take the survey now</a>
+														</td>
+													</tr>
+													<tr>
+														<td style="text-align:center;padding-top:0;padding-bottom:0;padding-right:60px;padding-left:60px;height:225px;">
+															<img alt="Picture of survey interface" width="382px" height="225px" src="/surveys/img/survey-snap.png" title="Survey" />
+														</td>
+													</tr>
+												</tbody>
+											</table>
+										</td>
+									</tr>
+								</tbody>
+							</table>
+
+							<table width="600" style="border-spacing:0;Margin:0 auto;">
+								<tbody>
+									<tr>
+										<td style="padding-top:0;padding-bottom:0;padding-right:0;padding-left:0;vertical-align:top;background-color:#eaecec;width:1px;">​</td>
+										<td width="600" bgcolor="#ffffff" style="padding-top:0;padding-bottom:0;padding-right:0;padding-left:0;vertical-align:top;Margin:0 auto;">
+											<table width="100%" style="border-spacing:0;">
+												<tbody>
+													<tr>
+														<td style="vertical-align:top;padding-bottom:30px;padding-left:60px;padding-right:60px;padding-top:30px;">
+															<table width="100%" style="border-spacing:0;">
+																<tbody>
+																	<tr>
+																		<td style="padding-top:10px;padding-bottom:0;padding-right:0;padding-left:0;vertical-align:top;text-align:center">
+																			<p style="Margin-top:0;font-weight:normal;color:#262626;font-family:sans-serif;font-size:22px;line-height:28px;Margin-bottom:15px;">The results of this survey help to inform how Ally can help improve communication and guidance around accessibility at your institution.</p>
```

Comment:
"The results of this survey will help inform how Ally can align with existing resources and processes and how it can most effectively allow instructors and students to reach out for help."